### PR TITLE
Modify Conda install instructions

### LINKED ISSUE #14681: Update Conda instructions
## Reason
Renamed, removed several conda packages which now need some documentation changes.

## Design
Modify the conda.md file to reflect necessary changes.

## Impact
None. The conda markdown file is not currently linked, and is only available by direct navigation.

### PR DESCRIPTION
**Pull request by milljm**
With the recent change to our conda package naming conventions, modify
the instructions to match.

Remove the bit about updating MOOSE (that is displayed when user continues
with the instructions).

Add a note about Catalina and Zsh conda installation issues.

Closes #14681

### CODE REVIEW

**Comment by milljm** [comment]
The conda install instructions can be found here: https://mooseframework.inl.gov/docs/PRs/14682/site/getting_started/installation/conda.html

File: modules/doc/content/getting_started/installation/conda.md [hunk 1]
```diff
@@ -19,10 +21,10 @@ Our preferred method for obtaining libraries necessary for MOOSE based Applicati
   conda config --add channels https://mooseframework.org/conda/moose
   ```
 
-- Install the moose-env package from Idaholab and name your environment 'moose':
+- Install the moose and moose-tools package from mooseframework.org, and name your environment 'moose':
 
   ```bash
-  conda create --name moose moose-env
+  conda create --name moose moose moose-tools
```
Comment (aeslaughter) [inline modules/doc/content/getting_started/installation/conda.md]:
```suggestion
  conda create --name moose moose-libmesh moose-tools
```

**Review by aeslaughter — CHANGES_REQUESTED** [review]

**Comment by gambka** [comment]
I can see the instructions are being updated but I did try the existing instructions yesterday.  I was able to add the moose channel as follows : 

`conda config --add channels https://mooseframework.org/conda/moose`

However, with that channel and the `conda-forge` channel added it would not find the `moose-env`.  Miniconda told me to look on the anaconda webpage for the environment since it could not be found in any of the current channels. On the webpage it lists as the channel for moose as idaholab.  Therefore, I tried 

`conda config --add channels idaholab`

after which the rest of the instructions worked without any issue.  Is the channel `https://mooseframework.org/conda/moose` or `idaholab`?

**Comment by milljm** [comment]
No one should be following these instructions just yet. They are not linked from any page. For the sneaky people that _do_ follow our conda PRs (much appreciated!), please know, if you wish to use these instructions, you should treat every install as if it was brand new (delete any previous moose environment, delete your ~/.condarc -or rather any of our custom channels, start over, etc).

So,
```bash
conda update --all
```
...is simply not something I am considering, when making changes to https://github.com/idaholab/conda-moose. That said, the mooseframework.org 'channel' should be relatively stable moving forward.

**Review by aeslaughter — APPROVED** [review]

**Pull request merged**